playbooks/roles/jumpbox/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+
+##########################
+# MYSQL
+##########################
+
+- name: Remove a cron job for mysql-backup
+  sudo: yes
+  cron:
+    name="Remove MySQL backup job"
+    cron_file=mysql_backup
+    state=absent
+
+# Print the shell tasks's stdout and stderr.
+- debug: var="bash_theme.stdout_lines + [ bash_theme.stderr ]"
+
+- name: Schedule MySQL backups
+  sudo: yes
+  cron:
+    name="Create MySQL backup job"
+    user="{{ os_admin_username }}"
+    cron_file=mysql_backup
+    special_time=daily
+    job="cd {{ oxa_tools_path }} && sudo {{ jumpbox_files_path }}/mysql_db_backup.sh --environment-file {{ env_file }} > ~/latest_sql_backup.log 2>&1"
+
+# Print the shell tasks's stdout and stderr.
+- debug: var="bash_theme.stdout_lines + [ bash_theme.stderr ]"
```

Comment:
bash_theme has not been registered within the task.  Suggest removing this statement OR register and rename identifier.